src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/CaseDataTagId.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+
+public class CaseDataTagId implements Serializable {
```

Comment:
Due to the composite key - we likely could get around this with not having a composite key but feels a good place as enforces no duplicate tags.